src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/draft/DraftClaimForm.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.civil.model.docmosis.draft;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.gov.hmcts.reform.civil.model.Address;
+import uk.gov.hmcts.reform.civil.model.ClaimAmountBreakupDetails;
+import uk.gov.hmcts.reform.civil.model.common.MappableObject;
+import uk.gov.hmcts.reform.civil.model.docmosis.common.EvidenceTemplateData;
+import uk.gov.hmcts.reform.civil.model.docmosis.common.Timeline;
+import uk.gov.hmcts.reform.civil.model.docmosis.lip.LipFormParty;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Builder
+@Getter
+@EqualsAndHashCode
+public class DraftClaimForm implements MappableObject {
+
+    private final LipFormParty claimant;
+    private final LipFormParty defendant;
+    private final Address claimantCorrespondenceAddress;
+    private final Address defendantCorrespondenceAddress;
+    private final String descriptionOfClaim;
+    private final List<Timeline> timelineEvents;
+    private final List<EvidenceTemplateData> evidenceList;
+    private final List<ClaimAmountBreakupDetails> claimAmount;
+    private final String totalClaimAmount;
+    private final String breakdownInterestRate;
+    private final String totalInterestAmount;
+    private final String howTheInterestWasCalculated;
+    private final String interestRate;
+    private final String interestExplanationText;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
```

Comment:
Hey is the date format the same everywhere? If it is why dont we config it globally instead of specifying it on every date field? It can be achieved with configOverride on ObjectMapper

Reply:
unfortunately no, there are yyyy-mm-dd , yyyy-MM-dd'T'HH:mm:ss also